### ISSUE TITLE
A simple "reload config" command

### ISSUE THREAD

**Issue by glozzom** [IlanCosman/tide#429]
When I open a new fish terminal, there is no prompt whatsoever.  This is something local to my machine likely, I will figure it out later.  But rather than use my preexisting config file, I have to generate a new config file for tide, and then everything is fine.  But I need to go through the whole process just to reload the config file.

I thought about how i3 window manager works, and how you can reload the config file in place, and if that were possibly I could just include that in my config.fish, and it will reload the config, and I get a prompt.   

So in summary, reloading the current config file in place would be nice, because I don't like hitting some sequence [1,2,2,1,2,3,3,2,2] just to have a prompt at all.

Any other solutions like directly addressing the problem would also be appreciated, but I've used worse workarounds. 

Reload from config with a simple included script.

**Comment by IlanCosman** [comment]
in v6
